cluster/node-pools/master-default/userdata.yaml

```diff
@@ -883,6 +883,15 @@ write_files:
           qps: {{ .Cluster.ConfigItems.event_rate_limit_config_qps }}
           burst: {{ .Cluster.ConfigItems.event_rate_limit_config_burst }}
 {{ end }}
+{{- if ne .NodePool.ConfigItems.sysctl_settings ""}}
+  - owner: root:root
+    path: /etc/sysctl.d/01-custom-sysctl-settings.conf
```

Comment:
I don't know what we have in the image by default, on my system it says
```
~$ cat /etc/sysctl.d/README 
This directory contains settings similar to those found in /etc/sysctl.conf.
In general, files in the 10-*.conf range come from the procps package and
serve as system defaults.  Other packages install their files in the
30-*.conf range, to override system defaults.  End-users can use 60-*.conf
and above, or use /etc/sysctl.conf directly, which overrides anything in
this directory.

After making any changes, please run "service procps start" (or, from
a Debian package maintainer script "invoke-rc.d procps start").
```
so I am wondering should we use `30-` or `60-` prefix to enable override of defaults as well?..

Reply:
There is also a symlink `99-sysctl.conf -> ../sysctl.conf`